client/extensions/wp-super-cache/cache-location.jsx

```diff
@@ -52,7 +53,7 @@ const CacheLocation = ( {
 								'/cache/ which translates to {{cacheLocation/}}',
 								{
 									components: {
-										cacheLocation: <span>{ cache_path || '' }</span>,
+										cacheLocation: <span>{ default_cache_path || '' }</span>,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14714/commits/b8909e61159afa262bc0ee1097206e01f5540ca3.